verification/src/http_server/handlers/verification/solidity/handlers.rs

```diff
@@ -0,0 +1,106 @@
+use crate::{
```

Comment:
rename this to `compile_and_verify` and remove this prefix from types, so we could use it like `compile_and_verify::Input` and etc.

also I'd like to rename it to something else, may be just `verify` or `verify_full` or something, but my ideas are worse, than current

Reply:
about naming
ok, basically this module verifies the contracts, but the main problem is that we're already have `verify` module or something
what's the difference then? you can say that this module compiles the code, but I think it is more like a implementation detail, and not the api description
instead, this module verifies the _source code_, and the other compares _bytecodes_
renaming that module is optional, but using this logic we could name this just `verify_source` or something
and then we'd say, that how exactly we transform source code into bytecode is impl detail